text/0121-config-service.md

```diff
@@ -0,0 +1,127 @@
+# A Dynamic Configuration Service for the SDK
```

Comment:
Consider to move this to `experimental/...` this way we know that it is experimental.

Reply:
Done! @SergeyKanzhelev, would it be a good idea to move the z-pages otep there as well?